tools/test-proxy/ci.yml

```diff
@@ -49,4 +49,11 @@ extends:
     - name: 'test_proxy_multiarch'
       dockerRepo: 'engsys/test-proxy'
       stableTags:
-      - 'latest'
\ No newline at end of file
+      - 'latest'
+    StandaloneExeMatrix:
+    - rid: win-x64
+      framework: net6.0
+    - rid: linux-x64
+      framework: net6.0
+    - rid: osx-x64
```

Comment:
Could we get `osx-arm64` too? I think one or two people on the JS team are using the new Apple Silicon Macs, that said I believe they should be able to run the `osx-x64` one anyway.

Reply:
Damn. The official RID list is wrong. Trying a couple different osX targets. They may just need to run the linux one :|

Reply:
I had a quick play around just now. If I remove the `net5.0` target framework from `Azure.Sdk.Tools.TestProxy.csproj` I can build an `osx-arm64` binary with this command:

```
dotnet publish --runtime osx-arm64 --framework net6.0 --self-contained true
```

I guess .NET 5 doesn't support ARM Mac binaries.

Reply:
BTW this isn't a blocker, more than happy to just ship x64 and revisit if the Mac folks run into issues!

Reply:
> I had a quick play around just now. If I remove the `net5.0` target framework from `Azure.Sdk.Tools.TestProxy.csproj` I can build an `osx-arm64` binary with this command:
> 
> ```
> dotnet publish --runtime osx-arm64 --framework net6.0 --self-contained true
> ```
> 
> I guess .NET 5 doesn't support ARM Mac binaries.

Excellent, I definitely should have remembered that .NET was part of the matrix.

Reply:
I only hadn't EoL-ed .NET 5 because .NET 6 isn't on all our boxes by default yet.

If I wasn't introducing a _guaranteed_ "install this thing" step, I'd drop support for .NET 5 in a heartbeat.

@weshaggard has anything changed with the story about putting .NET 6 on the agents by default?

Reply:
It looks like .NET 6 is on the image by default now https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md.